vendor/modules.txt

```diff
@@ -985,11 +985,12 @@ github.com/opencontainers/go-digest/digestset
 github.com/opencontainers/image-spec/identity
 github.com/opencontainers/image-spec/specs-go
 github.com/opencontainers/image-spec/specs-go/v1
-# github.com/opencontainers/runc v1.1.13
+# github.com/opencontainers/runc v1.1.14
 ## explicit; go 1.18
 github.com/opencontainers/runc/libcontainer/cgroups
 github.com/opencontainers/runc/libcontainer/configs
 github.com/opencontainers/runc/libcontainer/devices
+github.com/opencontainers/runc/libcontainer/system
```

Comment:
Unfortunately, this reintroduces `libcontainer/system` as a dependency, because we use the`libcontainer/cgroups`, which uses `libcontainer/utils.CleanPath` https://github.com/moby/moby/blob/5213a5afe79cfe18610e25fe34e85c90efdaf04f/vendor/github.com/opencontainers/runc/libcontainer/cgroups/file.go#L127

https://github.com/moby/moby/blob/5213a5afe79cfe18610e25fe34e85c90efdaf04f/vendor/github.com/opencontainers/runc/libcontainer/utils/utils.go#L61-L82